model/tensorflow_hub/examples/tfhub_text_classifier/accuracy.sh

```diff
@@ -0,0 +1,11 @@
+dffml accuracy \\
```

Comment:
All those these double backslashes should be single backslashes

```suggestion
dffml accuracy \
```